src/main/webapp/app/modules/conflicts/shared/components/location/locations-details.tsx

```diff
@@ -75,6 +80,18 @@ export class LocationsDetails extends React.Component<ILocationsDetailsProp, ILo
     }
   };
 
+  getLocationNumber = () => {
+    const { matchLocations, matchingLocation, locations } = this.props;
```

Comment:
Can we get a better distinguish in naming between matchLocations and matchingLocation?